docs/first_steps/first_steps.rst

```diff
@@ -160,6 +160,12 @@ The user can customize several aspects of Shapes Demo operation:
 
     + Push the *Start* button in order to resume Shapes Demo operation.
 
+- **Shared Memory Transport:** Fast DDS transport using Shared Memory Segment for Participants running in the same
+      host.
+
+- **Intraprocess:** Allow Intraprocess delivery when two Participants run in the same process.
+
+- **Data SHaring:** Allow Data Sharing delivery when two Participants run in the same process.
```

Comment:
```suggestion
- **Data Sharing:** Allow Data Sharing delivery when two Participants run in the same process.
```

I suggest adding a link to the Fast DDS documentation in all these new options so the user can have easy access to a more in-depth explanation of the option.

Reply:
:+1: